schema/defs.json

```
@@ -61,6 +61,12 @@
         }
       ]
     },
+    "base64": {
+      "type": "string",
+      "media": {
+        "contentEncoding": "base64"
```

Draft 4 changed this to "binaryEncoding": https://datatracker.ietf.org/doc/html/draft-luff-json-hyper-schema-00#section-4.3.1.1
```suggestion
        "binaryEncoding": "base64"
```

---

See https://github.com/opencontainers/image-spec/pull/1022#discussion_r1104938172, which links to a 2020 draft (newer than draft 4, as far as I can find) that disagrees :sweat_smile:

---

I wouldn't be opposed to updating to the newer drafts, but I think that's a bigger lift for us (double checking for any other changes, and updating the draft link in the various parts of the spec).

---

I'm being dragged into learning the json-spec stuff kicking and screaming after I changed all the `$schema` lines to https incorrectly. See #931 for fixing that mistake.